### DISCUSSION TITLE
The problem of Jacobian matrix w.r.t world frame with free type joints

### DISCUSSION THREAD

**Discussion by LHT-Liu** [google-deepmind/mujoco#2056] [Asking for Help]
### Intro

I am a student, I use MuJoCo for computing Jacobian matrix with free joint

### My setup

Apple M1 Pro macOS Sonoma 14.6.1

PyCharm 2023.3.5 (Community Edition)

mujoco 3.2.2

### My question

I want to calculate the Jacobian matrix of a  site position relative to the world coordinate system, but I think there is a problem with the result. I use `pinocchio` to perform the same calculation, but the results are not the same.

The first three lines are inconsistent, and the last three lines are consistent，why？

In **mujoco**

```
mujoco.mj_jacSite(m, d, Flange_site_jacp, Flange_site_jacr, Flange_site_id)
Flange_site_jacobian = np.vstack((Flange_site_jacp, Flange_site_jacr))  # vstack函数将多个数组沿垂直（行）方向堆叠的函数
print("带有floating joint的Flange_site处的雅可比矩阵（相对于world）: \n", Flange_site_jacobian)
```
![image](https://github.com/user-attachments/assets/67d4e5be-867c-4603-87f4-244065c00fa1)

In **pinocchio**

```
Flange_Jaco = pin.computeFrameJacobian(model, data, q, Flange_Id, pin.WORLD)
print("Flange_Jaco \n", Flange_Jaco)
```
![image](https://github.com/user-attachments/assets/33c59b3c-2667-4f87-8289-c6e67113c887)

@yuvaltassa
thanks

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by yuvaltassa** [comment]
This is not reproducible. Please make a simple, minimal model so that the model and your code together constitute a Minimal Reproducing Example.

**Reply by LHT-Liu** [reply]
this is reproducible after changing some files paths.

[Test_MM_FK_Jacobian.zip](https://github.com/user-attachments/files/17006174/Test_MM_FK_Jacobian.zip)


thanks @yuvaltassa
